### ISSUE TITLE
Delete UIEvent.cancelBubble() ?

### ISSUE THREAD

**Issue by hamishwillee** [mdn/content#4859]
I think we should delete [UIEvent.cancelBubble](https://developer.mozilla.org/en-US/docs/Web/API/UIEvent/cancelBubble) page (and mention of it in [UIEvent](https://developer.mozilla.org/en-US/docs/Web/API/UIEvent)).

There is no BCD for this, and it is marked as experimental and deprecated in MDN with the comment:

> This property is not part of any specification. Although the similar [Event.cancelBubble](https://developer.mozilla.org/en-US/docs/Web/API/Event/cancelBubble) property was included in an old Working Draft of W3C DOM Level 2. Microsoft has a description of it on [MSDN](https://msdn.microsoft.com/en-us/library/ms533545(v=vs.85).aspx).

The link on MSDN just links back to `Event.cancelBubble`. What this means is that we have a method that is non-spec and deprecated and since we don't have BCD there is no way for anyone to tell what browsers it might work on (though I guess maybe the old version of Edge based on the MSDN link). Further because it isn't in a spec or implemented anywhere that we know of there is no trigger to clean this up.

So my thinking is we should delete this method as "never going to be helpful for anyone".

@chrisdavidmills ?

**Comment by chrisdavidmills** [comment]
@hamishwillee yeah, I think I'd agree with this. It quite probably comes under the BCD [irrelevant features](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md#removal-of-irrelevant-features) guideline, and I know @foolip has bene making a bunchof removals based on that, getting rid of BCD and then removing the corresponding docs too.

**Comment by foolip** [comment]
This is more a case of a feature in the wrong place in the prototype chain than an irrelevant feature. It's [on `Event`](https://dom.spec.whatwg.org/#dom-event-cancelbubble) and https://developer.mozilla.org/en-US/docs/Web/API/Event/cancelBubble exists, so the UIEvent pages should just be redirected to that, and maybe some parts can be salvaged?
